scalableminds/webknossos
# Ensure that node positions are maintained as integers in front-end

Node positions are already stored as ints by the backend. This change ensures that the nodes are also ints in the front-end. The only exception is when currently moving a node as the this would lead to a weird UX, otherwise.

### URL of deployed dev instance (used for testing):
- https://intnodeposition.webknossos.xyz

### Steps to test:
- create and drag nodes
- disable interpolation
- zoom in and check that the nodes are at pixel corners

### Issues:
- fixes #5800 

------
(Please delete unneeded items, merge only when none are left open)
